libs/core/src/lib/checkbox/checkbox/checkbox.component.scss

```diff
@@ -3,3 +3,11 @@
 .fd-checkbox__text:empty {
     margin: 0;
 }
+
+.fd-checkbox,
+.fd-checkbox.is-warning,
+.fd-checkbox.is-error,
+.fd-checkbox.is-success,
+.fd-checkbox.is-information {
+    border-style: initial;
+}
```

Comment:
should go to fund-styles

Reply:
raised a PR https://github.com/SAP/fundamental-styles/pull/2881